eclipse_workflow.md

```diff
@@ -0,0 +1,305 @@
+---
+layout: index
+---
+
+# Developing NEST with Eclipse
+
+These instructions are based on Eclipse *Mars* (4.5). They are based
+on earlier instructions by Thomas Heiberg.
+
+## Installing Eclipse
+
+### Prerequisites
+
+You have to have a Java Development Kit (JDK) v. 1.7.0 (Java 7) or later
+installed. Just a Java Runtime Environment (JRE) is not enough.
+
+You can download the JDK from
+[Oracle's JDK8 Download Page](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html).
+
+### Debugger under OSX
+
+Under OSX, you need to install `gdb` to be able to debug NEST with
+Eclipse, since Eclipse does not support `lldb` (yet). For installation
+instructions, see
+
+ - [http://ntraft.com/installing-gdb-on-os-x-mavericks](http://ntraft.com/installing-gdb-on-os-x-mavericks)
+ - [https://sourceware.org/gdb/wiki/BuildingOnDarwin](https://sourceware.org/gdb/wiki/BuildingOnDarwin)
+
```

Comment:
Further you can use [Homebrew](http://brew.sh/)

    brew install homebrew/dupes/gdb